test/sanity/validate-modules/ignore.txt

```diff
@@ -339,7 +339,6 @@ lib/ansible/modules/database/postgresql/postgresql_ext.py E324
 lib/ansible/modules/database/postgresql/postgresql_lang.py E324
 lib/ansible/modules/database/postgresql/postgresql_schema.py E322
 lib/ansible/modules/database/postgresql/postgresql_schema.py E324
-lib/ansible/modules/database/postgresql/postgresql_user.py E322
 lib/ansible/modules/database/postgresql/postgresql_user.py E324
```

Comment:
E324 `Value for "default" from the argument_spec does not match the documentation`
E326 `Value for "choices" from the argument_spec does not match the documentation`

Could you please fix these while you are here?

Reply:
@gundalow , Great note!
I fixed all except role_attr_flags param of postgresql_user module because it's quite specific - trust me, better to leave it as it is ;)